docs/source/en/api/pipelines/panorama.mdx

```diff
@@ -60,6 +60,19 @@ and increase the VRAM usage.
 
 </Tip>
 
+<Tip>
+
+Circular padding is applied to ensure there are no stitching artifacts when working with 
+panoramas that needs to seamlessly transition from the rightmost part to the leftmost part. 
+By enabling circular padding (set `circular_padding=True`), the operation applies additional 
+crops after the rightmost point of the image, allowing the model to "see” the transition 
+from the rightmost part to the leftmost part. This helps maintain visual consistency in 
+a 360-degree sense and creates a proper “panorama” that can be viewed using 360-degree 
+panorama viewers. When decoding latents in StableDiffusion, circular padding is applied 
+to ensure that the decoded latents match in the RGB space.
```

Comment:
If possible would be also nice to link the comparison with and without circular padding. This way, the readers can spot-check the effectivity.

Reply:
Do you mean adding a text description of “normal” multi-diffusion, or adding a link to an image example of the results?

Reply:
> adding a link to an image example of the results?

Yeah just noting the difference with a comparative example like the one you showed in the PR description.

Reply:
That was a good idea indeed